libpimeval/src/pimResMgr.cpp

```diff
@@ -69,6 +69,7 @@ pimObjInfo::finalize()
   }
   m_numCoresUsed = coreIdCnt.size();
   m_numCoreAvailable = m_device->getNumCores();
+  m_isLoadBalanced = m_device->isLoadBalanced();
```

Comment:
while I think this is fine for now, in the future (when there's no imminent deadline), a good improvement would be to return an immutable device with properties